openblas-src/build.rs

```diff
@@ -104,6 +104,10 @@ fn build() {
     } else {
         cfg.no_static = true;
     }
+    cfg.target = match env::var("OPENBLAS_TARGET") {
+        Ok(target) => target.parse().ok(),
+        _ => None,
+    };
```

Comment:
I change here in e25b20fafcf67f8d5e867178c9eac839a3b1ca00 to immediately panic when `$OPENBLAS_TARGET` is invalid.